src/config/serverConfig.ts

```diff
@@ -1,5 +1,29 @@
-import { constants, ServerConfig } from '@projecttacoma/node-fhir-server-core';
+import { constants, ServerConfig, resolveSchema } from '@projecttacoma/node-fhir-server-core';
 import { MeasureService, LibraryService } from '../services';
+import capabilityStatementResources from './capabilityStatementResources.json';
+
+const customCapabilityStatement = (): fhir4.CapabilityStatement => {
+  const base_version = constants.VERSIONS['4_0_1'];
+  const CapabilityStatement = resolveSchema(base_version, 'CapabilityStatement');
+
+  return new CapabilityStatement({
+    id: 'MeasureRepositoryServiceCapabilityStatement',
+    name: 'MeasureRepositoryService',
+    title: 'FHIR Measure Repository Service Capability Statement',
+    status: 'active',
+    // date last modified
+    date: '2023-01-27',
```

Comment:
Interesting. I did see though that it is formatted like this in the Publishable Measure Repository service capability statement [here](https://build.fhir.org/ig/HL7/cqf-measures/CapabilityStatement-publishable-measure-repository.json.html). So this probably is correct.

Reply:
The `date` field in the capability statement is of type `dateTime` (more info [here](https://www.hl7.org/fhir/datatypes.html#dateTime)), and so I'm pretty sure both formats are technically correct. (I just did YYYY-MM-DD since I figured this is as precise as we need it to be.) I'll try the non-hyphenated format and see if it adds the hours/minutes/seconds automatically, but if it doesn't work then I feel like it's not a big deal and we can just use the hyphenated format for now 👍